### DISCUSSION TITLE
Library view on smartphone only possible in landscape with 20240818

### DISCUSSION THREAD

**Discussion by hjfisch** [rern/rAudio#1741] [Fixed Issues]
![raudio](https://github.com/user-attachments/assets/05573dc8-da18-451d-a8b8-1e55afe515ac)

**Comment by hjfisch** [comment]
![raudio2](https://github.com/user-attachments/assets/405164c2-4cdc-4c24-a023-5f995389f247)

**Comment by Editpo** [comment]
Same here
